conf/base/globals.yml

```
@@ -1,7 +1,8 @@
 n_qubits: 4
 n_layers: 22
 vqc_ansatz: "circuit_19"
-iec_ansatz: "spread_layers"
+iec_ansatz: "default" #spread_qubits
 data_reupload: 1.0
 learning_rate: 0.05
-shots: "None"
\ No newline at end of file
+shots: "None"
+output_interpretation: "all" # all, [qubit_index]
```

Same as above. You know documented the different values so you can skip other options here:

```suggestion
output_interpretation: "all"
```